server/server.js

```diff
@@ -15,6 +17,23 @@ export default function Server(connector, options = {}) {
 
   app.post("/batch", NotifyHandler);
   app.post("/notify", NotifyHandler);
+  app.get("/status", (req, res) => {
+    const { ship } = req.hull;
+    const messages = [];
+    let status = "ok";
+    if (!_.get(ship.private_settings, "code")) {
+      status = "error";
+      messages.push("Settings are empty");
+    }
+
+    const err = check(ship.private_settings.code);
```

Comment:
Where do you pass the objects (user, account etc...) - those aren't going to be defined and will trigger errors every time aren't they ?